openc3/lib/openc3/logs/packet_log_writer.rb

```diff
@@ -95,7 +95,12 @@ def write(entry_type, cmd_or_tlm, target_name, packet_name, time_nsec_since_epoc
 
       @mutex.lock if take_mutex
       begin
-        prepare_write(time_nsec_since_epoch, data.length, redis_topic, redis_offset, allow_new_file: allow_new_file)
+        if entry_type == :RAW_PACKET or entry_type == :JSON_PACKET
+          process_out_of_order = true
+        else
+          process_out_of_order = false
```

Comment:
entry_type can be `:TARGET_DECLARATION, :PACKET_DECLARATION, :RAW_PACKET, :JSON_PACKET, :OFFSET_MARKER, :KEY_MAP`. Can you add a comment about why we process `:RAW_PACKET, :JSON_PACKET` out of order but not with the rest.

Reply:
Will do